redisless/src/storage/mod.rs

```diff
@@ -18,6 +18,7 @@ pub trait Storage {
     fn read(&mut self, key: &[u8]) -> Option<&[u8]>;
     fn remove(&mut self, key: &[u8]) -> u32;
     fn contains(&mut self, key: &[u8]) -> bool;
+    fn value_type(&mut self, key: &[u8]) -> &[u8];
```

Comment:
I personally prefer the name `type_of` for this method but that's whatever

Reply:
yeah my first thought was `type_of(key)` that doesn't return the type of the key, but the type of the value inside it is confusing.